test/to_do_app/todos_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule ToDoApp.TodosTest do
```

Comment:
Ensure that the test files are working , use `mix test testFileName` to run the tests.